pkg/storkctl/clusterpair.go

```diff
@@ -344,17 +359,44 @@ func newCreateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptions
 			credentialData["type"] = []byte(provider)
 			credentialData["encryptionKey"] = []byte(encryptionKey)
 
-			ip, port, token, err := getClusterPairParams(dFile, dIP, dPort)
+			// Bail out if portworx-api service type is not loadbalancer type and the endpoints are not provided.
+			var err error
+			if len(sEP) == 0 {
+				srcPXEndpoint, err := getPXEndPointDetails(sFile)
+				if err != nil {
+					err = fmt.Errorf("unable to get portworx endpoint in source cluster. Err: %v", err)
+					util.CheckErr(err)
+					return
+				}
+				printMsg(fmt.Sprintf("source px endpoint is %s", srcPXEndpoint), ioStreams.Out)
```

Comment:
nit: Anything we are printing as part of the CLI output, lets start with upper case. 
```
Source PX endpoint is %s
```

Reply:
Done